bpo-41842: Add a unregister function in _codecs module (#22360)
python/cpython: Misc/NEWS.d/next/Library/2020-09-22-20-54-18.bpo-39337.zX2e5M.rst

> Add a :func:`_codecs.unregister` which could unregister a codec search
> function from the codec registry.

Reviewer: ```suggestion
Add :func:`codecs.unregister` function to unregister a codec search function.
```
